Neos.Flow/Classes/Security/Exception/AuthenticationRequiredException.php

```diff
@@ -22,4 +24,9 @@ class AuthenticationRequiredException extends \Neos\Flow\Security\Exception
      * @var integer
      */
     protected $statusCode = 401;
+
+    /**
+     * @var ActionRequest
+     */
+    public $interceptedRequest;
```

Comment:
I'm not 100% sure about this - but it is the best idea I have to transport the ActionRequest to the outside SecurityEntryPointMiddleware. Otherwise we need to do the work inside the dispatcher (see https://github.com/neos/flow-development-collection/pull/2223/files#diff-34a65e00b17dd586f553255a62368688485b195e8ef5cc1878222156ca1452bfR106-R108)